src/mugs.js

```diff
@@ -734,45 +734,23 @@ define([
         tagName: 'upload',
         icon: 'fcc fcc-fd-audio-capture',
         isODKOnly: true,
+        mediaType: "audio/*", /* */
         canOutputValue: false,
         writeCustomXML: function (xmlWriter, mug) {
-            var mediaType = mug.p.mediaType;
-            if (mediaType) {
-                xmlWriter.writeAttributeString("mediatype", mediaType);
-            }
-        },
-        init: function (mug, form) {
-            mug.p.mediaType = "audio/*"; /* */
+            xmlWriter.writeAttributeString("mediatype", mug.options.mediaType);
         },
-        spec: {
-            mediaType: {
-                lstring: 'Media Type',
-                visibility: 'visible',
-                presence: 'required'
-            }
-        }
     });
 
     var Image = util.extend(Audio, {
```

Comment:
Agreed. There should probably be a `Binary` base type that all of the media types inherit from.